.gitignore

```diff
@@ -1,5 +1,6 @@
 *.iml
 .idea
 .DS_Store
+build/
```

Comment:
There is no build folder at the root template. Not sure why do we need this?

<img width="666" alt="image" src="https://user-images.githubusercontent.com/16315358/202991043-9db14bd2-733c-4528-bc0f-995ee0f22fa6.png">

Reply:
@luongvo without `build/` the below auto-generated build folder will be added to git (this folder is generated every time we run the project)

![Screenshot 2565-11-21 at 15 18 38](https://user-images.githubusercontent.com/13492460/202999200-28030b44-e8db-4444-b61a-99401b46e277.png)

Reply:
@chornerman Do we have `buildSrc` folder at the root template? `buildSrc` is included in each `template-xml` or `template-compose` folder, not at the root level. So why do we need to ignore `build/` at root?

Reply:
@chornerman This gitignore file will actually take care of all the files in `template-compose` folder. 🤓 

<img width="997" alt="image" src="https://user-images.githubusercontent.com/16315358/203007754-85d2a2df-c0ca-4079-815a-d01f2210b169.png">

Reply:
Good point 😄 Let me update